templates/netlify/remix.init/server.js

```diff
@@ -0,0 +1,9 @@
+// Import path interpreted by the Remix compiler
+import * as build from "@remix-run/dev/server-build";
+import { createRequestHandler } from "@remix-run/netlify-edge";
+
+export default createRequestHandler({
+  build,
+  // process.env.NODE_ENV is provided by Remix at compile time
+  mode: process.env.NODE_ENV,
```

Comment:
Should we use `Deno` here instead, like we do in `deno` template?
CC/ @pcattori

https://github.com/remix-run/remix/blob/2248669ed59fd716e267ea41df5d665d4781f4a9/templates/deno/server.ts#L8

```suggestion
  mode: Deno.env.get("NODE_ENV"),
```

Reply:
I believe Remix uses `define` to set `"process.env.NODE_ENV"`, so it needs to be literally that string.

Reply:
https://github.com/remix-run/remix/blob/main/packages/remix-dev/compiler/compilerServer.ts#L107

Reply:
This would mean, the Deno template is wrong then?

CC/ @pcattori